osx/setup.sh

```diff
@@ -82,6 +82,14 @@ KA_LITE="ka-lite"
 KA_LITE_ZIP="$WORKING_DIR/$KA_LITE.zip"
 KA_LITE_DIR="$WORKING_DIR/$KA_LITE"
 
+
+PACKAGES_EXC="/usr/local/bin/packagesbuild"
+PACKAGES_PROJECT="$SCRIPTPATH/KA-Lite-Packages/KA-Lite-Monitor/KA-Lite.pkgproj"
+PACKAGES_BUILD_FOLDER="$SCRIPTPATH/KA-Lite-Packages/KA-Lite-Monitor/build"
+PACKAGES_DMG="Packages.dmg"
+PACKAGES_DMG_URL="http://s.sudre.free.fr/Software/files/$PACKAGES_DMG"
```

Comment:
Do we still need the Packages URL here if we have installed it on the build server directly?